packages/wdio-webdriver-mock-service/src/index.js

```diff
@@ -58,7 +58,7 @@ export default class WebdriverMockService {
         this.command.findElement().twice().reply(200, { value: elem2Response })
 
         this.command.elementClick(ELEMENT_ID).once().reply(200, { value: null })
-        this.command.elementClick(ELEMENT_ID).times(4).reply(500, { value: {
+        this.command.elementClick(ELEMENT_ID).times(1).reply(500, { value: {
```

Comment:
It only fails one time cause, the request handler doesn't retry on stale elements.

Reply:
@abjerstedt doesn't matter, fact is that if you would do the same request over and over again, you would always get stale element exception

Reply:
Reverted.